<!doctype html>
<meta charset=utf-32768>
<title>Test that the insertion point is defined in the error event of a parser-inserted script that actually started a fetch (but just had it fail).</title>
<script src=/resources/testharness.js></script>
<script src=/resources/testharnessreport.js></script>
<script>
  var t = async_test("");
  var writeDone = t.step_func_done(function(text) {
    assert_equals(text, "Some text");
  });
</script>
<iframe src="support/script-onerror-insertion-point-32766-helper.html"></iframe><iframe src="support/script-onerror-insertion-point-32766-helper.html"></iframe><iframe src="support/script-onerror-insertion-point-32767-helper.html"></iframe><iframe src="support/script-onerror-insertion-point--7778-helper.html"></iframe><iframe src="support/script-onerror-insertion-point-32766-helper.html"></iframe><iframe src="support/script-onerror-insertion-point-18446744075857035264-helper.html"></iframe>
